blog/2023-02-20-introducing-the-dhis2-java-sdk.md

```diff
@@ -0,0 +1,174 @@
+
+```xml
+<dependencies>
+    ...
+    <dependency>
+        <groupId>org.hisp.dhis.integration.sdk</groupId>
+        <artifactId>dhis2-java-sdk</artifactId>
+        <version>1.0.0</version>
+    </dependency>
+    ...
+</dependencies>
+```
+
+The version we’re referencing in the POM is `1.0.0` but you can always look up the latest version of the SDK from its GitHub repository [README](https://github.com/dhis2/dhis2-java-sdk/blob/main/README.md).
+
+Let’s take a deep dive into the SDK. Consider a standard integration app fetching data value sets from a DHIS2 instance and saving them to a different DHIS2 instance. Leveraging the DHIS2 Java SDK, the app's code to construct the DHIS2 client for the source instance is as follows:
+
+```java
+...
+import org.hisp.dhis.integration.sdk.Dhis2ClientBuilder;
+import org.hisp.dhis.integration.sdk.api.Dhis2Client;
+
+public class IntegrationApp {
+    public static void main( String[] args ) {
+        Dhis2Client sourceDhis2Client = Dhis2ClientBuilder.newClient( "https://source.dhis2.org/api", "d2pat_5xVA12xyUbWNedQxy4ohH77WlxRGVvZZ1151814092" ).build();
+    }
+}
+```
+
+The above is arguably trivial. `Dhis2ClientBuilder` builds a `Dhis2Client` to send requests to `https://source.dhis2.org/api` using the [PAT](https://docs.dhis2.org/en/use/user-guides/dhis-core-version-239/working-with-your-account/personal-access-tokens.html) `d2pat_5xVA12xyUbWNedQxy4ohH77WlxRGVvZZ1151814092` for authentication. 
+
+:::info
+`Dhis2ClientBuilder` can also build a `Dhis2Client` to authenticate with basic credentials as shown below:
+
+```java
+Dhis2ClientBuilder.newClient( "https://source.dhis2.org/api", "admin", "district" ).build();
+```
+:::
+
+The next line of code references the previous `sourceDhis2Client` to fetch the data value sets from the source DHIS2 server:
+
+```java
+DataValueSet dataValueSet = sourceDhis2Client.get( "dataValueSets" )
+    .withParameter( "dataSet", "BfMAe6Itzgt" )
+    .withParameter( "period", "202302" )
+    .withParameter( "orgUnit", "PLq9sJluXvc" )
+    .transfer()
+    .returnAs( DataValueSet.class );
+```
+
+Here's a breakdown of the method chain:
+
+* `sourceDhis2Client.get(...)` denotes the HTTP verb that will be used for the HTTP call. If we were to _POST_ a resource, the method invoked would be `sourceDhis2Client.post(...)`.
+
+* The parameter value inside the `get` method represents the URL path of the API endpoint (i.e., `dataValueSets`).
+
+* `withParameter(...)` sets a URL query parameter. In this example, we're requesting the data value set given the query parameters data set ID `BfMAe6Itzgt`, period `202302`, and organisation unit ID `PLq9sJluXvc`.
+
+* `transfer()` executes the HTTP call, that is, sends the request to the DHIS2 server.
+
+* `returnAs(...)` deserialises the HTTP response JSON body into a POJO of type `DataValueSet` since we are expecting a data value set from `https://source.dhis2.org/api/dataValueSets`.
+
+:::info
```

Comment:
This is a rather large "info" block, maybe it can be changed to a sub-header instead?